vcxagency-node/src/service/notifications/service-new-messages-unavailable.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const { ErrorFeatureDisabled } = require('../../errors/error-feature-disabled')
+
+module.exports.createServiceNewMessagesUnavailable = function createServiceNewMessagesUnavailable () {
+  function cleanUp () { }
+
+  async function hasNewMessage (_agentDid) {
+    throw ErrorFeatureDisabled('Feature is not enabled.')
+  }
+
+  async function registerCallback (_agentDid, _callbackId, _onNewMessageCallback) {
+    throw ErrorFeatureDisabled('Feature is not enabled.')
+  }
+
+  function cleanupCallback (agentDid, callbackId) { }
+
+  async function ackNewMessage (_agentDid) {
+    throw ErrorFeatureDisabled('Feature is not enabled.')
+  }
+
+  async function flagNewMessage (agentDid) { }
```

Comment:
I can't find some of these functions used anywhere - where are they used? (Also, there is inconsistent syntax in unused arguments - `agentDid` vs `_agentDid`)

Reply:
This is implementation of `ServiceNewMessages` for enterprise agency. It will throw `ErrorFeatureDisabled` on some calls resulting in HTTP response of  status 409.

Reply:
The method is called from `wireUpApplication` function